src/Config/Form/FormOptionsBag.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Bolt\Extension\Bolt\BoltForms\Config\Form;
+
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+/**
+ * Form options configuration for BoltForms
+ *
+ * Copyright (c) 2014-2016 Gawain Lynch
```

Comment:
You wrote it, so i think this should have your name on it. ;-)

Reply:
No, the copyright is a global, the `@author` is the correct one to use here and has Arthur's attribution

Reply:
Scratch that.. I was thinking of `@author`, which _does_ have your name on it.. 

Must be the lazy-saturday-afternoon-beers. 🍻

Reply:
Just to add this for the record (as details were just now discussed in person), the reason being that any licence change needs the written concent of *all* copyright holders … now the GPL will keep this FOSS licenced, but we don't know what the future holds and might what to switch to the WTFPL or similar :stuck_out_tongue_winking_eye: at some stage

Reply:
> Must be the lazy-saturday-afternoon-beers. :beers:

I'll 'ave a 'alf